### DISCUSSION TITLE
This library is wonderful ❤️

### DISCUSSION THREAD

**Discussion by abenhamdine** [Comcast/react-data-grid#3410] [General]
Just to say that.
I have thoroughly tested different datagrid libraries, because we have a very data-intensive application we progressively migrate to Next.js, with some tables with more of tens of thousands rows.

This library is so far the fastest we have used 🚀 
It seems like virtualization is very well implemented. 

The keyboard accessibility is also great, it's very rare.

The library is a bit low level (you have to implement the filters by yourself for example) but frankly it's not a problem.

Congrats to the authors/maintainers ! 👏

**Comment by merabtenei** [comment]
@abenhamdine I agree, there's still some missing features like freezing a column to the right side. At first We went with [ReactGrid](https://reactgrid.com/examples/?example=liquidity-planner) which is also a great library, but we didn't like the sticky behavior of the input in edit mode (when scrolling while editing).

**Comment by Ben-CA** [comment]
It takes a bit to learn, but I've been quite happy with how it works for the basics.
